mst/migrate-to-mst.md

```diff
@@ -0,0 +1,123 @@
+---
+title: Migrating an existing TimescaleDB database to Managed Service for TimescaleDB
+excerpt: Migrating an existing TimescaleDB database to Managed Service for TimescaleDB
+product: mst
+keywords: [migrate, databse]
+tags: [ingest, backup, restore]
+---
+
+# Migrate the entire database to Managed Service for TimescaleDB
+
+You can migrate your data from self-hosted TimescaleDB to Managed Service for
+TimescaleDB and use exclusive features, including separate scaling for compute
+and storage requirements, first-class multi-node support, and more.
```

Comment:
Separate scaling and multi-node are Timescale Cloud-exclusive features. As far as I'm aware, they're not available on MST.

Reply:
> Separate scaling and multi-node are Timescale Cloud-exclusive features. As far as I'm aware, they're not available on MST.

Let me confirm, this as well.

Reply:
Hi @alexthinath  could you please let me know the features that are exclusive to MST.

Reply:
yes Charis's comment is correct. To be a bit more specific, on Cloud you can choose Storage and Compute completely independently. There is a slider bar for each resource. On MST, you can only choose a "plan" which is a combination of CPU+Storage. E.g. 100GB storage + 2 CPU. From there you can size up the disk amount, but you're still choosing a "base plan" that couples the two resources.